doc/classes/Object.xml

```diff
@@ -369,7 +369,8 @@
 			<argument index="1" name="default" type="Variant" default="null" />
 			<description>
 				Returns the object's metadata entry for the given [code]name[/code].
-				Throws error if the entry does not exist, unless [code]default[/code] is not [code]null[/code] (in which case the default value will be returned).
+				Throws error if the entry does not exist, unless [code]default[/code] is not [code]null[/code] (in which case the default value will be returned). See also [method has_meta], [method set_meta] and [method remove_meta].
+				[b]Note:[/b] Metadata whose [code]name[/code] starts with an underscore ([code]_[/code]) is considered editor-only. It will not be available for reading in an exported project.
```

Comment:
> It will not be available for reading in an exported project.

This is not true. We aren't removing it yet, so I'd remove this part until this data is properly stripped.